packages/cli/src/commands/serve.js

```diff
@@ -1,43 +1,116 @@
 import fs from 'fs'
 import path from 'path'
 
+import chalk from 'chalk'
+import execa from 'execa'
 import terminalLink from 'terminal-link'
 
-import { getPaths } from '../lib'
+import { getPaths, getConfig } from '../lib'
 import c from '../lib/colors'
+import { isTypeScriptProject } from '../lib/project'
 
 export const command = 'serve [side]'
 export const description = 'Run server for api or web in production'
 
-export const builder = async (yargs) => {
-  const {
-    apiCliOptions,
-    webCliOptions,
-    commonOptions,
-    apiServerHandler,
-    webServerHandler,
-    bothServerHandler,
-  } = await import('@redwoodjs/api-server')
+export function coerceRootPath(path) {
+  // Make sure that we create a root path that starts and ends with a slash (/)
+  const prefix = path.charAt(0) !== '/' ? '/' : ''
+  const suffix = path.charAt(path.length - 1) !== '/' ? '/' : ''
+
+  return `${prefix}${path}${suffix}`
+}
 
+export const builder = async (yargs) => {
   yargs
     .usage('usage: $0 <side>')
     .command({
       command: '$0',
       descriptions: 'Run both api and web servers',
-      handler: bothServerHandler,
-      builder: (yargs) => yargs.options(commonOptions),
+      handler: async (argv) => {
+        const serverFileName = `server.${isTypeScriptProject() ? 'ts' : 'js'}`
+        const serverFilePath = path.join(getPaths().api.dist, serverFileName)
+        if (fs.existsSync(serverFilePath)) {
+          console.log(
+            `${chalk.hex('#ff845e')(
+              `------------------------------------------------------------------\n 🧪 ${chalk.green(
+                'Experimental Feature'
+              )} 🧪\n------------------------------------------------------------------`
+            )}`
+          )
+          console.log(
+            `Using the experimental API server file: 'api/dist/${serverFileName}'.`
+          )
+          console.log(
+            `${chalk.hex('#ff845e')(
+              '------------------------------------------------------------------'
+            )}\n`
+          )
+          await execa('node', [path.join('dist', serverFileName)], {
+            cwd: getPaths().api.base,
+            stdio: 'inherit',
+            shell: true,
+          })
```

Comment:
Is there's anything wrong with using that `cwd` option and then `path.join('dist', serverFileName)`?

Reply:
Note: use `yarn node` here.